cmd/arcstat.in

```diff
@@ -569,6 +598,16 @@ def calculate():
     v["el2mru"] = d["evict_l2_eligible_mru"] // sint
     v["el2inel"] = d["evict_l2_ineligible"] // sint
     v["mtxmis"] = d["mutex_miss"] // sint
+    v["ztotal"] = (d["stream_hits"] + d["stream_future"] + d["stream_stride"] +
+                   d["stream_past"] + d["stream_misses"]) // sint
+    v["zhits"] = d["stream_hits"] // sint
+    v["zfuture"] = (d["stream_future"] + d["stream_stride"]) // sint
```

Comment:
Since you are adding two values here but also expose one separately, wouldn't it worth to expose another separately too to make it more obvious?